site/docs/latest/admin/http.md

```diff
@@ -0,0 +1,317 @@
+---
+title: BookKeeper http endpoints
+---
+
+This document is to introducing BookKeeper http endpoints that could be used for BookKeeper administration.
+If user want to use this feature, parameter "httpServerEnabled" need to be set to "true" in file conf/bk_server.conf.
```

Comment:
To use this feature, set "httpServerEnabled" to "true" in file conf/bk_server.conf.

Reply:
Thanks, wll change it.